README.md

```diff
@@ -1,20 +1,25 @@
-# Rust on Android
+> ⚠️ **Can't find what you're looking for?** ⚠️
```

Comment:
I don't like quotes for this... but learned last week that there's a way to get a specially highlighted block:

https://github.com/community/community/discussions/16925

Reply:
Yeah, it's mainly just so it gets formatted distinctly like a notice. Using the `> **Note**` markdown extension is probably better though.

Reply:
We could use either:

> **Note**

> **Warning**

I like the new format regardless :)